### DISCUSSION TITLE
List of known ETF instances

### DISCUSSION THREAD

**Discussion by MarcoMinghini** [INSPIRE-MIF/helpdesk-validator#761] [Show and tell]
Dear all,
we would be interested in collecting and maintaining a list of known ETF instances - no matter if public or private, and no matter if they are testing INSPIRE resources or not.

Therefore, if you manage/use or if you are aware of an ETF instance, can you please share the following information?

* Country and responsible organisation
* Status (experimental, pre-operational, operational)
* URL (if public)
* ETF version used
* Type of deployment (e.g. build from source, or using Docker)
* Comments (e.g. what it allows to test, who is using it)

Thank you,
Marco

**Comment by zguo** [comment]
We have an internal ETF testing instance: 

Country and responsible organisation: US, esri
Status (experimental, pre-operational, operational): experimental
URL (if public): internal URL
ETF version used: 1.0.8
Type of deployment (e.g. build from source, or using Docker): manual deployment
Comments (e.g. what it allows to test, who is using it): we use it for validation test of ArcGIS for INSPIRE metadata and services

Thanks,

ZhongPing

**Comment by thijsbrentjens** [comment]
For the Netherlands we have an instance running:

Country and responsible organisation: Netherlands, geonovum
    Status: operational
    URL (if public): via https://validatie.geostandaarden.nl/
    ETF version used: 1.x in production, migration to 2.x is due
    Type of deployment: deployment of the war-file in Tomcat
    Comments: tests for services, metadata and data for INSPIRE TGs and for Dutch profiles and datamodels, including geometry. The INSPIRE validators will be phased out, in favor of the reference validator.

**Comment by ghost** [comment]
Hi @MarcoMinghini ,

thanks a lot for raising the question. We also have ETF instance in our federal state:

Country:                                  
Federal state Baden-Württemberg (BW) in Germany 

Status: 
operational

Responsible organization:       
State Office for Spatial Information and Land Development Baden-Württemberg

URL:                                      
https://www.geoportal-bw.de/validator/

ETF version:                           
ETF WebApp 2.0.0-b190926T1239

Type of deployment:
build from source

Comments:                             
It allows to test metadata records against metadata profile SDI-BW. It is used by SDI-Community in Baden-Würrtemebrg.

**Comment by alitka** [comment]
In Germany we started to develop a new national validator.

- Country and responsible organisation: Germany, BKG (on behalf of GDI-DE)
- Status: experimental (under development scheduled release: 1. quarter 2020)
- URL: not yet available (under development, see comment under status) 
- ETF version used: 2.x
- Type of deployment: Docker
- Comments:
  - the current development for the new version of GDI-DE Testsuite will include (all) ETF-tests of the INSPIRE Reference Validator
  - current INSPIRE-tests of GDI-DE Testsuite will be replaced
  - national extensions based on ETF are imaginable but not yet concrete (currently, national tests are based on TEAM Engine)

**Reply by g-weber** [reply]
> Status Update
> 
   * Country and responsible organisation: Germany, BKG (on behalf of GDI-DE)
   * Status: operational since 2. Quarter 2020
   * URL:  https://testsuite.gdi-de.org/
   * ETF version used: Within the GDI-DE Testsuite we are updating always the [latest release](https://github.com/INSPIRE-MIF/helpdesk-validator/releases) for our integrated ETF-Validator Instances  

   * Type of deployment: The GDI-DE Testsuite is installed as a Kubernetes-Cluster using OGC Team Engine and ETF-Validator as Testengines. In the Cluster there are actually integrated three (number is scalable) instances of the ETF-Validator (based on Dockerimage) in paralell
   * Comments:

     - the GDI-DE Testsuite supports all tests of the latest version of ETF-Validator and locally developped tests of GDI-DE members.
     - national tests are based on TEAM Engine
     - free access for all users of the GDI-DE Community, and other data and/or service providers

**Comment by iuriemaxim** [comment]
Hi @MarcoMinghini ,

Country:
Romania

Status:
operational

Responsible organization:
National Meteorological Administration of Romania (ANM)

URL:
https://inspire.meteoromania.ro/etf-webapp

ETF version:
https://github.com/inspire-eu-validation/community/releases/tag/v1.0.8

Type of deployment:
deployment of the war-file in Tomcat

Comments:
Used for testing and debugging metadata, data and services of ANM, especially when the official ETF is down or too busy.
